src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -61,7 +61,7 @@ if peripheral.find("printer") then
 end
 table.insert(tMenuItems, "Exit")
 
-local sStatus = "Press Ctrl to access menu"
+local sStatus = "Press Ctrl or click here to access menu"
```

Comment:
On normal computers, this should probably still show "Press Ctrl to access menu" - we don't have mouse support there!

Reply:
Double check how the message looks on advanced pocket computers, might be worth trying to shorten it

E.g. changing 'access' to 'open'

Reply:
Pocket computers have a shorter message anyway:

https://github.com/SquidDev-CC/CC-Tweaked/blob/da419b24e701229eee3afa06315969c5d2a4f972/src/main/resources/assets/computercraft/lua/rom/programs/edit.lua#L65-L67

Reply:
Perfect